lib/Listener/TextRecognizedListener.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (c) 2020 Robin Windey <ro.windey@gmail.com>
+ *
+ *  @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace OCA\WorkflowOcr\Listener;
+
+use OCA\WorkflowOcr\Events\TextRecognizedEvent;
+use OCP\EventDispatcher\Event;
+use OCP\EventDispatcher\IEventListener;
+use Psr\Container\ContainerInterface;
+use Psr\Log\LoggerInterface;
+
+class TextRecognizedListener implements IEventListener {
```

Comment:
I think this class should not be added to this app. It's rather a sample for an app which listens to the `TextRecognizedEvent` so maybe it would be better to add this as a sample to the `README.md` inside a new section like "Events emitted by this app" or something similar?

Reply:
yeah my bad, this was just for testing purposes and ended up in the commits by error 🙈

Reply:
Removed :heavy_check_mark: